relenv/common.py

```diff
@@ -14,6 +14,10 @@
 import urllib.error
 import urllib.request
 
+
+# relenv package version
+__version__ = "0.0.1"
```

Comment:
Opposed to use something that get's the version from the tag?
Instead of always having to manually update this? Specially since we'll tag from the UI?

Reply:
Yes, I'm using setup tools for this.

Reply:
@dwoz why not?
This means you have to update that, and then kick off the build from the UI?

It will happen that, at some point in the future, someone will run the build from the UI and forget to update this....